internal/pkg/template/env.go

```diff
@@ -0,0 +1,74 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package template
+
+import (
+	"bytes"
+	"fmt"
+)
+
+// Paths of service cloudformation templates under templates/services/.
+const (
+	// EnvCFTemplatePath is the path where the cloudformation for the environment is written.
+	EnvCFTemplatePath     = "environment/cf.yml"
+	envNestCFTemplatePath = "environment/nest/cf/%s.yml"
```

Comment:
What do you think of this path instead? `"environment/cf/%s.yml"` 
I feel like it conforms to `services/common/cf/%s.yml` better.